### ISSUE TITLE
Function of Pre_Eular_Integral 

### ISSUE THREAD

**Issue by sauradip** [Pokerlishao/LoopGaussian#4]
Hi Authors, 

I am curious to know what is the function of this line 

https://github.com/Pokerlishao/LoopGaussian/blob/50ce0861462911172990431e65a13a966bfe37eb/train_loop.py#L110

and what does this mean ? 

https://github.com/Pokerlishao/LoopGaussian/blob/50ce0861462911172990431e65a13a966bfe37eb/train_loop.py#L117

**Comment by Pokerlishao** [comment]
Hi~,

This function is used to pre-compute the Euler integral. It calculates both the forward and backward positions of each Gaussian point at each frame within the Eulerian motion field. The final position of each Gaussian point is then determined by interpolation, as described by the formula: $\hat{p}_i(t)=\alpha p_i(t)+(1-\alpha)p_i(t-T)$, where $\alpha=\frac{t}{T}$ (see Section 4.5 in the original paper).

I saved this computation to render the video using the original 3D-GS renderer.

If you have any further questions, feel free to ask!

Best regards,

**Comment by sauradip** [comment]
Thanks for the clairifcation. 

I want to know the extent of control i can get from the 3 hyperparams in this line 

https://github.com/Pokerlishao/LoopGaussian/blob/50ce0861462911172990431e65a13a966bfe37eb/train_loop.py#L107

i saw some dramatic movements by changing these parameters.

**Comment by Pokerlishao** [comment]
Hi~,

Thank you for your question!

These three parameters control the magnitude of motion on each axis. The appropriate values for these parameters can vary depending on the specific scenario, which may need to be re-tested for different scenarios to get a satisfactory visualization. 

Based on my experience,  you can directly set these parameters to approximately $1e-3$ of the scene's resolution `clu_resolution` is generally a good starting point. You may need to fine-tune them for different scenes to get the desired results.
